### ISSUE TITLE
exporting to A4 format

### ISSUE THREAD

**Issue by mdg453** [olup/notable#20]
hey, at my university they are only accepting A4 size pages, and the regular boox note taking app don't have this option, if you can add this option it will really help me ^^.
thank you,
meitar.
also if you can add colors for the new colored devices it will also help a lot!, thank you so much!

**Comment by olup** [comment]
You mean, when exporting a page or book ? How to you see the conversion in size, fit or fill with white ?

**Comment by mdg453** [comment]
yes a notebook,  I don't know what its the current page size of the pdf, i think its depends of the original size of the paper, but since its adapting size, its most likely wont be  A4 size (A4 size in pixels at 300 DPI: 2480 x 3508 pixels.) so my best thinking of solution is to make an option for the app to have A4 fixed size, from the paper.
If you are able to do that' it would be grate!
